game/models.py

```
@@ -136,4 +162,30 @@ class Meta:
 
     def __str__(self):
         return f"{self.user.username} - {self.achievement.title}"
-    
\ No newline at end of file
+
+class FeaturedBadge(models.Model):
```

Ruff (0.15.15)

[warning] Model does not define `__str__` method

(DJ008)